libs/core/src/lib/product-switch/product-switch/product-switch.component.ts

```diff
@@ -1,21 +1,19 @@
-import { ChangeDetectionStrategy, Component, ViewEncapsulation, Input, HostBinding } from '@angular/core';
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
 import {Placement} from 'popper.js';
 import { BasePopoverClass } from '../../popover/base/base-popover.class';
 
 @Component({
     selector: 'fd-product-switch',
     templateUrl: './product-switch.component.html',
```

Comment:
Why are you removing the ViewEncapsulation? Our components have encapsulation set to `None`.